pkgs/servers/asterisk/default.nix

```diff
@@ -1,16 +1,16 @@
 { stdenv, lib, fetchurl, fetchsvn,
-  jansson, libxml2, libxslt, ncurses, openssl, sqlite,
+  jansson, libedit, libxml2, libxslt, ncurses, openssl, sqlite,
   utillinux, dmidecode, libuuid, newt,
   lua, speex,
-  srtp, wget, curl, iksemel
+  srtp, wget, curl, iksemel, pkgconfig
 }:
 
 let
   common = {version, sha256, externals}: stdenv.mkDerivation rec {
     inherit version;
     name = "asterisk-${version}";
 
-    buildInputs = [ jansson libxml2 libxslt ncurses openssl sqlite utillinux dmidecode libuuid newt lua speex srtp wget curl iksemel ];
+    buildInputs = [ jansson libedit libxml2 libxslt ncurses openssl sqlite utillinux dmidecode libuuid newt lua speex srtp wget curl iksemel pkgconfig ];
```

Comment:
`pkgconfig` belongs to `nativeBuildInputs`. It would not hurt to add line breaks in this list.

Reply:
`utillinux` is probably also a candidate for `nativeBuildInputs`.